redisworks/redisworks.py

```diff
@@ -128,6 +128,8 @@ def get_obj(value, actual_type):
             value = datetime.datetime.strptime(value, DATE_FORMAT).date()
         elif actual_type in {dict, list} or isinstance(actual_type, (MutableMapping, Iterable)):
             value = json.loads(value)
+        elif actual_type is bool:
+            value = actual_type == "True"
```

Comment:
I'm not sure if I understand why this fixes the issue. If `actual_type is bool` then it is never equal to "True" string, right?